examples/listinvoices/src/main.rs

```diff
@@ -0,0 +1,58 @@
+use lnd;
+use std::fs;
+use std::env;
+use std::thread;
+
+#[tokio::main]
+async fn main() {
+
+    //Get the macaroon and cert files
+    let macaroon = fs::read("admin.macaroon").unwrap();
+    let cert = fs::read("tls.cert").unwrap();
+
+    //Get the url connection string of the lnd node
+    let env_lnd_url = env::var("LND_URL");
+    if env_lnd_url.is_err() {
+        println!("The $LND_URL environment variable could not be found.\n");
+        std::process::exit(1);
+    }
+    let node_address = "https://".to_owned() + env_lnd_url.unwrap().as_str();
+
+    //Make the connection to LND
+    let mut lightning;
+    match lnd::Lnd::connect_with_macaroon(node_address.clone(), &cert, &macaroon).await {
```

Comment:
Could you assign to the lightning variable from the match instead of `let mut`?
For example `let lightning = match ...`. It's not a requirement, but I think it looks more like usual rust.

Reply:
Sure, I can make those changes.